packages/liveblocks-react-ui/src/primitives/Composer/index.tsx

```diff
@@ -427,11 +569,12 @@ function ComposerEditorPlaceholder({
 const ComposerMention = forwardRef<HTMLSpanElement, ComposerMentionProps>(
   ({ children, asChild, ...props }, forwardedRef) => {
     const Component = asChild ? Slot : "span";
+    const { isFocused } = useComposer();
     const isSelected = useSelected();
 
     return (
       <Component
-        data-selected={isSelected || undefined}
+        data-selected={(isFocused && isSelected) || undefined}
```

Comment:
Is this something we missed previously? Does this mean mentions inside a composer are focusable?

Reply:
This change is a remnant of when I tried to fix LB-1374 as part of this PR, since I'll fix the problem in another PR I removed this from this one for now.